packages/resolver/lib/sources/npm.ts

```diff
@@ -68,6 +60,25 @@ export class NPM implements ResolverSource {
     return { body, filePath: import_path };
   }
 
+  resolveAndParse(basePath: string, packageName: string, contractName: string) {
```

Comment:
Do you think this should be a utility method outside of this class since it is really not standard in the same way `require` and `resolve` are?

Reply:
I think it's OK as it is -- yeah I agree it's not great, but it's not like this is going to spread any further, at least for now.  Rule of three, not rule of two, right? :)  IDK, @gnidan may have stronger opinions on this. :P